sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/ParseOptionsHelper.cs

```diff
@@ -31,18 +31,26 @@ namespace SonarAnalyzer.UnitTest.TestFramework
 {
     internal static class ParseOptionsHelper
     {
-        private static readonly IEnumerable<ParseOptions> defaultParseOptions =
-            ImmutableArray.Create<ParseOptions>(
-                new CS.CSharpParseOptions(CS.LanguageVersion.CSharp7),
-                new CS.CSharpParseOptions(CS.LanguageVersion.CSharp7_1),
-                new CS.CSharpParseOptions(CS.LanguageVersion.CSharp7_2),
-                new CS.CSharpParseOptions(CS.LanguageVersion.CSharp7_3),
-                new VB.VisualBasicParseOptions(VB.LanguageVersion.VisualBasic12),
-                new VB.VisualBasicParseOptions(VB.LanguageVersion.VisualBasic14),
-                new VB.VisualBasicParseOptions(VB.LanguageVersion.VisualBasic15),
-                new VB.VisualBasicParseOptions(VB.LanguageVersion.VisualBasic15_3),
-                new VB.VisualBasicParseOptions(VB.LanguageVersion.VisualBasic15_5)
-                );
+        private static readonly IEnumerable<ParseOptions> defaultParseOptions;
+
+#pragma warning disable S3963 // The static fields are dependent between them so the values cannot be set inline
```

Comment:
This seems to be a false positive on our side :)

Reply:
do we have a ticket for it?